.github/actions/spelling/patterns.txt

```diff
@@ -0,0 +1,107 @@
+# This file contains various spellcheck patterns for @check-spelling/check-spelling
+# and is used in the spellcheck workflow. See ../../workflows/spellcheck.yml for more information.
+
+# marker to ignore all code on line
+^.*/\* # no-spell-check-line \*/.*$
+# marker for ignoring a comment to the end of the line
+// # no-spell-check.*$
+# marker for ignoring all lines with disabled spellcheck
+.*spellcheck: disable
```

Comment:
Why not just:
```suggestion
*spellcheck: disable
```
The `.` makes no sense, as you follow it with a `*`.

Reply:
That's in glob patterns, not regular expressions. In order for that to be a valid regular expression, the `*` needs to define a repeating character, which in this case is `.`, a special token in RegEx that represents every letter.